### ISSUE TITLE
Cannot install latest version on Craft 3.7.25.1

### ISSUE THREAD

**Issue by pvaessen** [enupal/translate#52]
### Description
We need to install Enupal Translate v 2.3.0 but getting lots of errors.

This is the error log:
```
composer require enupal/translate:2.3.0 -W
Info from https://repo.packagist.org: #StandWithUkraine
./composer.json has been updated
Running composer update enupal/translate --with-all-dependencies
Loading composer repositories with package information
Info from https://repo.packagist.org: #StandWithUkraine
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires craftcms/cms 3.7.25.1 -> satisfiable by craftcms/cms[3.7.25.1].
    - Conclusion: don't install guzzlehttp/guzzle 7.1.1 (conflict analysis result)
    - Conclusion: don't install guzzlehttp/guzzle 7.2.0 (conflict analysis result)
    - Conclusion: don't install guzzlehttp/guzzle 7.3.0 (conflict analysis result)
    - Conclusion: don't install guzzlehttp/guzzle 6.5.5 (conflict analysis result)
    - Conclusion: don't install guzzlehttp/guzzle 7.4.0 (conflict analysis result)
    - Conclusion: don't install guzzlehttp/guzzle 7.4.1 (conflict analysis result)
    - Conclusion: don't install guzzlehttp/guzzle 7.4.2 (conflict analysis result)
    - enupal/translate 2.3.0 requires viniciusgava/google-translate-api 3.1.0 -> satisfiable by viniciusgava/google-translate-api[3.1.0].
    - Root composer.json requires enupal/translate 2.3.0 -> satisfiable by enupal/translate[2.3.0].
    - Conclusion: don't install guzzlehttp/guzzle 7.1.0 (conflict analysis result)
    - akamai-open/edgegrid-client is locked to version 1.0.0 and an update of this package was not requested.
    - viniciusgava/google-translate-api 3.1.0 requires guzzlehttp/guzzle ^7.0.1 -> satisfiable by guzzlehttp/guzzle[7.0.1, ..., 7.4.x-dev (alias of dev-master)].
    - You can only install one version of a package, so only one of these can be installed: guzzlehttp/guzzle[dev-master, 5.3.3, 5.3.4, 5.3.x-dev, 6.0.0, ..., 6.5.x-dev, 7.0.0-beta.1, ..., 7.4.2].
    - craftcms/cms 3.7.25.1 requires guzzlehttp/guzzle ^6.5.5|^7.2.0 -> satisfiable by guzzlehttp/guzzle[6.5.5, 6.5.x-dev, 7.2.0, ..., 7.4.x-dev (alias of dev-master)].
    - guzzlehttp/guzzle 7.4.x-dev is an alias of guzzlehttp/guzzle dev-master and thus requires it to be installed too.
    - Conclusion: don't install guzzlehttp/guzzle dev-master (conflict analysis result)
```

Installation failed, reverting ./composer.json and ./composer.lock to their original content.

### Steps to reproduce

1. composer require enupal/translate:2.3.0 -W
2.

### Additional info

- Craft version: 3.7.25.1
- PHP version: 7.3.1
- Database driver & version: MySQL 8.0.27
- Plugin version: 2.2.1

**Comment by andrelopez** [comment]
Hi @pvaessen it looks like you have a composer conflict with the `akamai-open/edgegrid-client` lib (currently requires Guzzle 6.x) however it looks like there is a PR opened to add support to Guzzle 7 [here](https://github.com/akamai/AkamaiOPEN-edgegrid-php-client/pull/39) when that PR is merged and `akamai-open/edgegrid-client` release a new version you'll be ready to update the plugin version.

**Comment by pvaessen** [comment]
We removed the akamai-open/edgegrid-client lib and now we are able to upgrade to v2.3.0
When we now try to open Enupal Translate --> Translations, no matter which country we select, we always are getting this error:
"A server error occurred".
Nothing is logged in the .log files.
Is there a way to figure out what this error is ?
We upgraded from v2.1.1 to v2.3.0 thinking that this would solve our problem.
However, on v2.1.1 we also see this "A server error occurred".

The screen is always empty and we have a lot of entries in our site.php files, for example /translations/de-DE/site.php file
